submissions/ArthurGorbenko/Friend's app/js/script.js

```diff
@@ -0,0 +1,282 @@
+const CONSTS = {
+  LIST_OF_CARDS: document.querySelector(".list"),
+  DIRECT_MESSAGE_FORM: document.querySelector(".direct-message"),
+  FORM: document.querySelector(".form"),
+  SEND_BUTTON: document.querySelector(".send-on-click-button"),
+  SEARCH_FIELD: document.querySelector(".search-field"),
+  RADIO_AGE_FORM: document.querySelector(".wrapper-radio-age"),
+  RADIO_GENDER_FORM: document.querySelector(".wrapper-radio-gender"),
+  EMAIL_FIELD: document.querySelector(".email-field"),
+  TEXT_AREA: document.querySelector(".textarea"),
+  ALERT_MESSAGE: document.querySelector(".alert-message"),
+  RESET_BUTTON: document.querySelector(".reset"),
+  HAMBURGER: document.querySelector(".hamburger.hamburger--slider"),
+  THEME_WRAPPER: document.querySelector(".wrapper-theme-change"),
+  BODY: document.querySelector(".default"),
+  MENU: document.querySelector(".menu"),
+  WRAPPER_MENU: document.querySelector(".wrapper-menu")
+};
+
+const MOBILE_RESOLUTION = "(max-width: 599px)";
+const SORT_BY_ASCENDING = "ascending";
+const CARD_CLASS = "card";
+const BUTTON_CLASS = "button";
+const DARK_INPUT_ID = "dark";
+
+let filteredByGender = false;
+let radioButtons = document.querySelectorAll(".visually-hidden");
+let usersConst;
+let users;
+
+const letterToUpperCase = word => {
+  return word.charAt(0).toUpperCase() + word.slice(1);
+};
+
+const refreshDMWindow = () => {
+  CONSTS.FORM.classList.remove("visually-hidden");
+  CONSTS.ALERT_MESSAGE.classList.add("visually-hidden");
+  CONSTS.TEXT_AREA.value = "";
+};
+
+let menuToggled = false;
```

Comment:
It will be better to have all variable declaration in the top of file or function. Please grab it into one place. 

> Declarations on Top
It is a good coding practice to put all declarations at the top of each script or function.
This will:
Give cleaner code
Provide a single place to look for local variables
Make it easier to avoid unwanted (implied) global variables
Reduce the possibility of unwanted re-declarations

Reply:
Agree.Done.